drivers/spi/spi_handlers.c

```diff
@@ -11,15 +11,14 @@
 /* This assumes that bufs and buf_copy are copies from the values passed
  * as syscall arguments.
  */
-static void copy_and_check(struct spi_buf_set *bufs,
-			   struct spi_buf *buf_copy,
-			   int writable)
+static struct spi_buf_set *copy_and_check(struct spi_buf_set *bufs,
+			   struct spi_buf *buf_copy, int writable)
```

Comment:
indent the parameters to last '(' + 1 space (and keep writable parameter on its own line as well)

Reply:
@tbursztyka updated.

Reply:
thanks